bitrise.yml

```diff
@@ -102,6 +102,7 @@ workflows:
 
             echo "bitrise_xcode"
             COUNT_XCUI=$(./test-fixtures/generate-metrics.sh /Users/vagrant/git/xcodebuild_fennec.log all)
+            cat /Users/vagrant/git/xcodebuild_fennec.log
```

Comment:
Just curious why do we need this?

Reply:
I was trying to debug the lint warning/error.